lib/FileFinder.js

```diff
@@ -18,19 +18,74 @@ var fs = require('fs');
 var path = require('path');
 
 /**
+ * Wrapper for options for a find call
+ * @class
+ * @param  {Object}         options                 Parameter bag of configuration options.
+ * @param  {Array.<String>} options.scanDirs        Directories to scan, ex: ['html/']
+ * @param  {Array.<String>} options.extensions      Extensions to searc for, ex: ['.js']
+ * @param  {function|null}  options.ignore          Optional function to filter out paths
+ * @param  {bool}           options.useNative       Use os find instead of javascript's
+ * @param  {bool}           options.followSymlinks  Whether to follow symlinks.
+ * @param {Object} options
+ */
+function FileFinder(options) {
+  this.scanDirs = options && options.scanDirs || ['.'];
+  this.extensions = options && options.extensions || ['.js'];
+  this.ignore = options && options.ignore || null;
+  this.useNative = options && options.useNative || false;
+  this.followSymlinks = options && options.followSymlinks || true;
```

Comment:
This should default to false for backwards compatibility. Was testing this alongside jest.